central/group/datastore/internal/store/store_impl.go

```diff
@@ -166,13 +166,13 @@ func updateInTransaction(tx *bolt.Tx, group *storage.Group) error {
 	id := group.GetProps().GetId()
 	buc := tx.Bucket(groupsBucket)
 
-	// Check whether the to-be-added group is a default group, ensure that it does not yet exist.
-	defaultGroupExists, err := checkDefaultGroupForProps(tx, group.GetProps())
+	defaultGroup, err := getDefaultGroupForProps(tx, group.GetProps())
 	if err != nil {
 		return err
 	}
 
-	if defaultGroupExists {
+	// Only disallow update of a default group if it does not update the existing default group, if there is any.
+	if defaultGroup != nil && defaultGroup.GetProps().GetId() != id {
```

Comment:
Should this block be _after_ the id block below (in case it's an update by props)?

Reply:
Yup, you're right. Posted #2569 to fix this.